src/main/kotlin/uk/gov/justice/digital/hmpps/subjectaccessrequestworker/services/SubjectAccessRequestWorkerService.kt

```diff
@@ -81,6 +81,7 @@ class SubjectAccessRequestWorkerService(
       append("failed with error: ${exception.message}")
     }
     log.error(errorMessage, exception)
+    exception.printStackTrace()
```

Comment:
Should this need to be in a log entry?

Reply:
I removed this originally but added it back in because we don't get the full stack trace from the exception log.